lib/commons/reusable_container.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+
+class ReusableContainer extends StatelessWidget {
+  const ReusableContainer(
+      {super.key, required this.functionName, required this.iconName});
+  final Function() functionName;
+  final IconData iconName;
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        const SizedBox(width: 12),
+        GestureDetector(
+          onTap: functionName,
+          child: Container(
+              height: 35,
+              width: 35,
```

Comment:
height, width, border radiuses and paddings should be multiple of 4 for screen responsiveness and good practices